components/swatch/stories/swatch.stories.js

```diff
@@ -22,33 +28,218 @@ export default {
 		},
 		rounding: {
 			name: "Rounding",
+			description: "The amount of corner rounding for a swatch.",
+			defaultValue: "regular",
 			type: { name: "string" },
 			table: {
 				type: { summary: "string", required: true },
 				category: "Component",
+				defaultValue: { summary: "Regular", },
```

Comment:
```suggestion
				defaultValue: { summary: "regular", },
```

I think the cases need to match here.